modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -205,9 +209,13 @@ export class {{classname}} {
             body: formParams.toString(),
 {{/hasFormParams}}
             search: queryParameters,
-            withCredentials:true
+            withCredentials:isWithCredentials
```

Comment:
Got it!  :)
So I could remove all code about the local var:  
```
let isWithCredentials: boolean = false;
if (this.configuration.withCredentials) {
  isWithCredentials = this.configuration.withCredentials;
}
```
and just use this expression to pass the value of configuration to the option. is that right?

sorry, I am Java developer and habitually `null` checker.